tests/integration/test_xrdsst.py

```diff
@@ -0,0 +1,134 @@
+
+import unittest
+import time
+import subprocess
+import os
+import json
+import git
+import urllib3
+
+from definitions import ROOT_DIR
+from xrdsst.controllers.base import BaseController
+from xrdsst.controllers.init import InitServerController
+from xrdsst.controllers.timestamp import TimestampController
+from xrdsst.controllers.token import TokenController
+from xrdsst.main import XRDSSTTest
+
+
+class TestXRDSST(unittest.TestCase):
+    git_repo = 'https://github.com/nordic-institute/X-Road.git'
+    local_folder = os.path.join(ROOT_DIR, "tests/integration/X-Road")
+    branch_name = 'develop'
+    docker_folder = local_folder + '/Docker/securityserver'
+    dist = 'bionic-6.25.0'
+    image = 'xroad-security-server:' + dist
+    url = 'https://localhost:4000/api/v1/api-keys'
+    roles = '[\"XROAD_SYSTEM_ADMINISTRATOR\",\"XROAD_SECURITY_OFFICER\"]'
+    header = 'Content-Type: application/json'
+    file_name = 'api-key.txt'
+    max_retries = 100
```

Comment:
Little shorter timeout probably possible?

Reply:
Made the timeout shorter